docs/configuration.md

```diff
@@ -653,11 +653,8 @@ Options:
       --preview
           Enable preview mode; enables unstable formatting. Use `--no-preview`
           to disable
-      --range-start <RANGE_START>
-          Format code starting at the given character offset (zero based)
-      --range-end <RANGE_END>
-          Format code ending (exclusive) at the given character offset (zero
-          based)
+      --range <RANGE>
+          Format code in the given <RANGE>
```

Comment:
Hmmm should this say more about what `<RANGE>` is? (I know this is the auto-generated docs. It looks like it's generated from a Rust doc comment? And that comment appears to have more details...)

Reply:
I suspect it only takes the first paragraph?

Reply:
Okay, I have no idea what it does. It also removes all terminating `.`.. ugh

Reply:
The script takes the short help only. I reworded the documentation so that the short help already provides some details on `<RANGE>`